tap_google_sheets/transform.py

```diff
@@ -167,15 +167,35 @@ def transform_sheet_decimal_data(value, sheet_title, col_name, col_letter, row_n
         return col_val
 
 # transform number values in the sheet
-def transform_sheet_number_data(value, sheet_title, col_name, col_letter, row_num, col_type):
-    if type(value) == int:
-        return int(value)
-    elif type(value) == float:
-        return transform_sheet_decimal_data(value, sheet_title, col_name, col_letter, row_num, col_type)
+def transform_sheet_number_data(formatted_value, unformatted_value, sheet_title, col_name, col_letter, row_num, col_type):
+    # formatted_value: the value in the google sheet as it is displayed
+    # unformatted_value: the value converted into serial number as per Google API
+    if type(unformatted_value) == int:
+        try:
+            # handle US number type format ie. 123,456.10 -> 123456.10
+            numeric_value = formatted_value.replace(",", "")
+            # verify we can convert formatted value to float
+            # for scientific formatted numbers:
+            #   formatted value: "1.23E+03"
```

Comment:
What does this mean? If it is an example do mention it before.

Reply:
This is an example. Added comment to recognize it as example.